core/iwasm/libraries/lib-socket/test/manifest.json

```diff
@@ -0,0 +1,3 @@
+{
+    "name": "WASI lib-socket tests"
```

Comment:
I'd call them WAMR lib-socket tests, there's no such thing (yet) as WASI sockets (and even if there was, it's [very different](https://github.com/WebAssembly/wasi-sockets) to the WAMR implementation).